neo/SmartContract/WillCardContainer.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+namespace Neo.SmartContract
+{
+    public class WillCardContainer<T> : IReadOnlyList<T>
```

Comment:
`WildCardContainer`?

Reply:
WillCardArray?

Reply:
What is WillCard? I think it's wildcard.

Reply:
xD my mistake